docs: clarify suite releases vs dropin updates for forked implementation (#1241)
hlxsites/aem-boilerplate-commerce: README.md

### Updating your drop-in packages

The only thing you need to actively track after forking is your **npm dependencies** — specifically the `@dropins/*` and `@adobe/*` packages (including `@adobe/magento-storefront-event-collector` and `@adobe/magento-storefront-events-sdk`) listed in your `package.json`. These packages follow semantic versioning. Minor and patch releases are non-breaking by contract, so routine updates should be safe to apply.

To check which packages have newer versions available:

```bash
npm outdated
```

To update a specific drop-in to its latest stable release:

```bash
npm install @dropins/storefront-cart@latest
npm run postinstall
```

Always run `postinstall` after any drop-in update — it copies the built assets from `node_modules` into `scripts/__dropins__`, which is what Edge Delivery Services serves. Note that `npm` does not run `postinstall` automatically when you install a specific package, so this step must always be done manually.

### Automated dependency PRs

> This repo includes a GitHub Actions workflow that runs every Monday and automatically opens a pull request when newer stable versions of `@adobe/*` or `@dropins/*` packages are available. This works similarly to Dependabot or Renovate. Once you fork the repo, this workflow runs in your fork and keeps a PR queue of available updates for you to review and merge at your own pace.

Author: Alludes to #1239 . Do not merge this README change until that PR is merged.

Also consider changing specific verbiage like "runs every Monday", since that could change. 
```suggestion
This repo includes a GitHub Actions workflow that can automatically open a pull request when newer versions of `@adobe/*` or `@dropins/*` packages are available, within your defined [semver](https://semver.org/) range. This works similarly to Dependabot or Renovate. Once you fork the repo, this workflow runs in your fork and keeps a PR queue of available updates for you to review and merge at your own pace.
```
